src/servers/ZoneServer2016/managers/smeltingmanager.ts

```diff
@@ -282,6 +282,14 @@ export class SmeltingManager {
     subEntity: CollectingEntity,
     container: LoadoutContainer
   ) {
+    const currentDate = new Date(server.inGameTimeManager.time * 1000);
+    const currentHour = currentDate.getUTCHours();
+    const isRaining =
+      server.weatherManager.rainingHours.includes(currentHour) ||
+      server.weatherManager.globalPrecipation > 1 ||
```

Comment:
why not just using .rain ? and also this way it's inconsistent it could rain for 4 mins and if the rain stop and the smeltablemanager run right after it won't ack that it was raining